morphomuseum.csl

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>MorphoMuseuM</title>
+    <id>http://www.zotero.org/styles/morphomuseum</id>
+    <link href="http://www.zotero.org/styles/morphomuseum" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-harvard" rel="template"/>
+    <link href="http://morphomuseum.com/instructionsToAuthors" rel="documentation"/>
+    <category citation-format="author-date"/>
```

Comment:
You already have `<category citation-format="author-date"/>` on line 15, so please delete line 9.